chain-core/src/init/config.rs

```diff
@@ -80,7 +80,7 @@ impl fmt::Display for DistributionError {
 
 /// Initial configuration ("app_state" in genesis.json of Tendermint config)
 /// TODO: reward/treasury config, extra validator config...
-#[derive(Debug, PartialEq, Eq, Clone)]
+#[derive(Debug, PartialEq, Eq, Default, Clone)]
```

Comment:
why are defaults added to chain-core?

Reply:
there seems to be an error when deserializing something if not add Default, I remember, I'll check again.